.github/workflows/build.yml

```diff
@@ -109,6 +110,7 @@ jobs:
         name: "Build supervisord image"
         needs: build-php
         runs-on: ubuntu-latest
+        if: github.repository_owner == 'pimcore'
```

Comment:
```suggestion
        if: github.repository == 'pimcore/docker'
```
Since we're using repo secrets it's probably better to check for the repo name 🤔